TECHNOLOGY.md

```diff
@@ -0,0 +1,34 @@
+# YouSee DNA
+YouSee DNA provides the building blocks to assemble consistent user interfaces across yousee.dk. It's a CSS framework and a resource library. On [dna.yousee.dk](https://dna.yousee.dk) you'll find comprehensive guides and components for building YouSee websites and applications.
+
+
+## Languages & Frameworks
+| Name        | Status  | NPS | Comments |
+|-------------|---------|-----|----------|
+| HTML5       | Adopted | 10  |          |
+| Sass (SCSS) | Adopted | 9   |          |
+| Bootstrap   | Adopted | 7   | Used for the flexbox grid only |
+| JavaScript  | Adopted | 10  |          |
+| Handlebars  | Adopted | 5   | All components are developed in handlebars |
+| Nunjucks    | Adopted | 5     | Only used to override "Fractal code" as static page exporter |
+
+## Platforms

```

Comment:
Dunno if we should add github pages here? but :shrug:

Reply:
I was not sure either 🤔The documentation says:

> Platforms. Things that we build software on top of such as mobile technologies like Android, virtual platforms like the JVM, or generic kinds of platforms like hybrid clouds.